caffe2/contrib/fakelowp/test/test_sls_8bit_nnpi_fp16.py

```diff
@@ -41,7 +42,7 @@ def Skip_test_SLS_NonQuantized_fp16(self):
         fp16_c2_net = core.Net("test_fp16_c2")
         fp16_c2_net.SparseLengthsSumFakeFP16AccFP16(["D", "I", "L"], "fp16_out")
 
-        input_dict = {}
+        input_dict : Dict[Any, Any] = {}
```

Comment:
Why do we need this extra typing info? Does not seem to be informative.

Reply:
Simply to avoid mypy complaining. Adding an `Any` annotation is still more useful than adding `# type: ignore`.